main.go

```diff
@@ -108,21 +136,13 @@ func reinitDatabase(database *sql.DB, cfg config.Config) (err error) {
 			log.Println("Add category", taskCategory.Name)
 		}
 
-		if task.NameEn == "" {
-			task.NameEn = task.Name
-		}
+		checkTaskNameEn(&task)
 
-		if task.Name == "" {
-			task.Name = task.NameEn
-		}
+		checkTaskName(&task)
 
-		if task.DescriptionEn == "" {
-			task.DescriptionEn = task.Description
-		}
+		checkTaskDescriptionEn(&task)
 
-		if task.Description == "" {
-			task.Description = task.DescriptionEn
-		}
+		checkTaskDescriprion(&task)
```

Comment:
Seems like "reduce only for reduce" :)

Probably if you just move all fallback's for english description to separate function like a "fillEnglishFallback" it may be better than that.

Reply:
Is it okey to separate functions on "fillEnglishFallback" and "fillRussianFallback"?

Reply:
I think that the single function `fillTranslateFallback` will be ok.

Reply:
done

Reply:
done

Reply:
done